app/src/main/java/org/simple/clinic/cvdrisk/CVDRiskModule.kt

```diff
@@ -31,4 +38,19 @@ class CVDRiskModule {
   fun lastPullToken(rxSharedPrefs: RxSharedPreferences): Preference<Optional<String>> {
     return rxSharedPrefs.getOptional("last_cvd_risk_pull_token", StringPreferenceConverter())
   }
+
+  @Provides
+  @AppScope
+  @OptIn(ExperimentalStdlibApi::class)
+  fun cvdRiskCalculationSheet(moshi: Moshi, configReader: ConfigReader): CVDRiskCalculationSheet? {
```

Comment:
So, this happens on the main thread. Since this JSON is large, I am worried that it might cause ANR. How about we load it lazily and in a different thread and place where we use it?